lang/cs/Org.Apache.REEF.IO/FileSystem/Hadoop/HadoopFileSystem.cs

```diff
@@ -142,6 +142,12 @@ public void CreateDirectory(Uri directoryUri)
             _commandRunner.Run("dfs -mkdir " + directoryUri);
         }
 
+        public bool IsDirectory(Uri uri)
+        {
+            var stdOut = _commandRunner.Run("dfs -count " + uri).StdOut;
+            return stdOut.Any() && stdOut.First().StartsWith("1");
```

Comment:
>.StartsWith("1"); [](start = 49, length = 17)

Elaborate this check. What if it the result is "11"?

Reply:
fixed

---
In reply to: [194555273](https://github.com/apache/reef/pull/1473#discussion_r194555273) [](ancestors = 194555273)